src/ui/pages/SendTransaction/TransactionConfiguration/TransactionConfiguration.tsx

```diff
@@ -101,26 +203,71 @@ export function TransactionConfiguration({
           </UIText>
         </HStack>
       ) : (
-        <NetworkFeeLine
-          configuration={configuration}
-          onConfigurationChange={onConfigurationChange}
-          transaction={transactionWithFrom}
-          chain={chain}
-          keepPreviousData={keepPreviousData}
-          onFeeValueCommonReady={onFeeValueCommonReady}
-        />
+        <div
+          style={{
+            ['viewTransitionName' as string]: `network-fee-default-line-${id}`,
+          }}
+        >
+          <NetworkFeeLine
+            configuration={configuration}
+            onConfigurationChange={onConfigurationChange}
+            transaction={transactionWithFrom}
+            chain={chain}
+            keepPreviousData={keepPreviousData}
+            onFeeValueCommonReady={onFeeValueCommonReady}
+          />
+        </div>
       )}
       {preferences?.configurableNonce ? (
-        <NonceLine
-          userNonce={configuration.nonce}
-          transaction={transactionWithFrom}
-          chain={chain}
-          onChange={
-            onConfigurationChange
-              ? (nonce) => onConfigurationChange({ ...configuration, nonce })
-              : null
+        <div style={{ ['viewTransitionName' as string]: `nonce-line-${id}` }}>
+          <NonceLine
+            userNonce={configuration.nonce}
+            transaction={transactionWithFrom}
+            chain={chain}
+            onChange={
+              onConfigurationChange
+                ? (nonce) => onConfigurationChange({ ...configuration, nonce })
+                : null
+            }
+          />
+        </div>
+      ) : null}
+      {FEATURE_LOYALTY_FLOW === 'on' ? (
```

Comment:
Same here... looks like we could just check the truthiness of `FEATURE_LOYALTY_FLOW`, no need to `=== 'on'`

Reply:
https://github.com/zeriontech/zerion-wallet-extension/pull/653/files#r1834131020